From 7d6bc6105ed2f405fbb683a5d6ceafc55e92e3c0 Mon Sep 17 00:00:00 2001
From: drizzt <timothy.redaelli@gmail.com>
Date: Wed, 18 Mar 2015 16:01:27 +0100
Subject: [PATCH] Don't include usb.core in btchipPersoWizard.py

It's useless and it enforce pyusb to be a mandatory depends, but on not-Windows it's not needed.
---
 btchip/btchipPersoWizard.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/btchip/btchipPersoWizard.py b/btchip/btchipPersoWizard.py
index 3220227..486e6d9 100644
--- a/btchip/btchipPersoWizard.py
+++ b/btchip/btchipPersoWizard.py
@@ -28,7 +28,6 @@
 except:
 	MNEMONIC = False
 
-from usb.core import USBError
 from btchipComm import getDongle, DongleWait
 from btchip import btchip
 from btchipUtils import compress_public_key,format_transaction, get_regular_input_script
